recipes/sox/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: sox
+  version: 14.4.2
+
+source:
+  fn: sox-14.4.2.tar.gz
+  url: http://sourceforge.net/projects/sox/files/sox/14.4.2/sox-14.4.2.tar.gz
+  md5: d04fba2d9245e661f245de0577f48a33
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+
+about:
+  home: http://sox.sourceforge.net/
+  license: LGPL
```

Comment:
Wow, thanks for all the prompt and awesome feedback. I will incorporate
into the PR (probably just focus on getting one working, probably just Lua
for now, then submit separate PRs for each project. There will eventually
be a few dozen...)

On Wed, Mar 30, 2016 at 3:18 PM Amir Mohammadi <notifications@github.com>
wrote:

> In recipes/sox/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/230#discussion_r57885882>
> :
>
> > +package:
> > +  name: sox
> > +  version: 14.4.2
> > +
> > +source:
> > +  fn: sox-14.4.2.tar.gz
> > +  url: http://sourceforge.net/projects/sox/files/sox/14.4.2/sox-14.4.2.tar.gz
> > +  md5: d04fba2d9245e661f245de0577f48a33
> > +
> > +build:
> > +  number: 0
> > +  detect_binary_files_with_prefix: true
> > +
> > +about:
> > +  home: http://sox.sourceforge.net/
> > +  license: LGPL
>
> I also have these lines in my recipe for a successful build on Linux:
>
> requirements:
>   build:
>     - libpng
>     - zlib
>     - xz
>   run:
>     - libpng
>     - zlib
>     - xz
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/230/files/4e5e1b4ef169544ecf51cb6a5385163f7bf4874c#r57885882>
>